plugin.xml

```diff
@@ -53,9 +53,7 @@
         </js-module>
     </platform>
 
-    <!-- ios -->
```

Comment:
hm, we have those all through the plugins I think.
not really necessary when you look at the next line `<platform name="ios">` 🤷‍♀

Reply:
Yeah I know... would like to omit the discussion regarding comments and if the are a code smell or not, when a comment is fine, ... 😅 - but in this case it's pretty what to expect from `<platform name="ios">`

Reply:
Nah, I'm good. Just used to it because it is part of the Cordova docs as well. Those lines have been there for... (check) ... only 2 years. That's not long enough to get nostalgic 🔥